Iceberg-Plugin.package/IceGitHubPullRequestModel.class/instance/obtainAvailableBranchNames.st

```diff
@@ -0,0 +1,8 @@
+private
+obtainAvailableBranchNames
+	^ ((self repository allBranches 
```

Comment:
I think this is weird, because the same list of branch names is used for both base and head repositories, and there is no warranties that branches might exists in both sides. Isn't it possible to change the set of branches accordingly to the selected repository?